### ISSUE TITLE
Switch icons disappearing after loading save

### ISSUE THREAD

**Issue by Lazzyman53** [Lewisk3/UltimateClassicMinimap#6]
Each time a new save is loaded, I need to reapply having the switches show up on the mini-map.

**Comment by Vortaka** [comment]
Yes, this is true...  However, I found a way to put them back in (with some little tests!)
After you load a game, go in Option/UCMinimap Options/Switch/Apply
And they are right back!
So, I guess an easy dirty quick fix would be after each load to do a switch apply...

P.S.  I was even able to bind a key to the "apply" (netevent ucm.switches.apply).  If I knew this scripting language more, I might be able to make it auto launch on load or something...  Alas, bind is the best I can do right now...
